src/main/java/seedu/address/ui/StudentDetailsPanel.java

```diff
@@ -52,6 +54,7 @@ public StudentDetailsPanel(Student student, int displayedIndex) {
         name.setText(student.getName().fullName);
         phone.setText(student.getPhone().value);
         email.setText(student.getEmail().value);
+        remark.setText(student.getRemark());
```

Comment:
If there is no remark given to the student, would it be better to indicate "no remark given" on the UI or leave the Remark field blank?

Reply:
I left it blank because I thought it would be more intuitive that way